Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -487,6 +492,116 @@ customerIssueCarousalView  push =
     ]
 
 
+customerIssuePopupView :: forall w. (Action -> Effect Unit) ->  CustomerIssue -> PrestoDOM (Effect Unit) w
+customerIssuePopupView  push config = 
+      let cardsArray = config.customerIssueCards
+          title = case (head cardsArray) of
```

Comment:
Do this case match at the top level and then use that variable to access its property instead of doing the case match multiple times